pkg/ocm/config.go

```diff
@@ -39,6 +42,46 @@ var URLAliases = map[string]string{
 	"crc":         "https://clusters-service.apps-crc.testing",
 }
 
+func ValidOCMUrlAliases() []string {
+	keys := make([]string, 0, len(URLAliases))
+	for k := range URLAliases {
+		keys = append(keys, k)
+	}
+	return keys
+}
+
+// URL Precedent (from highest priority to lowest priority):
+//  1. runtime `--env` cli arg (key found in `urlAliases`)
+//  2. runtime `--env` cli arg (non-empty string)
+//  3. config file `URL` value (non-empty string)
+//  4. sdk.DefaultURL
+//
+// Finally, it will try to url.ParseRequestURI the resolved URL to make sure it's a valid URL.
+func ResolveGatewayURL(optionalParsedCliFlagValue string, optionalParsedConfig *config.Config) (string, error) {
```

Comment:
This is a nitpick, but generally in CS/Rosa, we now use CamelCase for acronyms. So this would be `ResolveGatewayUrl`